pyext/src/exhaust.py

```diff
@@ -181,19 +187,22 @@ def main():
                 args.path, idfile_A, idfile_B)
     else:
         args.extension = "rmf3"
+        rmsd_custom_ranges = precision_rmsd.parse_custom_ranges(args.selection)
```

Comment:
This needs to be protected from running when `args.selection` is `None`, as it will fail otherwise. (This is why the existing test cases are broken.)

Reply:
thanks for taking the time, i am sorrt this was not quite ready... Now this should be taken care of

Reply:
For the tests.. I am not familiar with that, perhaps @tanmoy7989 can put it in? should just be copying density_ranges.txt and enabling --selection